src/common/types/value.cpp

```diff
@@ -39,6 +40,19 @@ Value::Value(string val) : type(TypeId::VARCHAR), is_null(false) {
 	}
 }
 
+//void Value::ValidateString() {
```

Comment:
Please remove the commented code, at least for the final PR ;)

Reply:
Okay.